transifex/native/cds.py

```diff
@@ -27,6 +27,19 @@
 }
 
 
+def EtagStore(object):
+    """ Manges etags """
+
+    def __init__(self):
+        self._mem = {}
+
+    def set(self, key, value):
+        self._mem[key] = value
+
+    def get(self, key):
+        return self._mem.get(key, '')
+
```

Comment:
Probably we need to a duration policy here

Reply:
Add this suggestion as a comment so it outlives this PR.